packages/trader/src/AppV2/Components/TradeParameters/__tests__/trade-parameters.spec.tsx

```diff
@@ -17,7 +17,7 @@ const TRADE_PARAMS = {
     ACCUMULATORS_INFORMATION: 'AccumulatorsInformation',
     MULTIPLIER: 'Multiplier',
     RISK_MANAGEMENT: 'RiskManagement',
-    MULTIPLIERS_INFORMATION: 'MultipliersInformation',
+    MULTIPLIERS_DEAL_CANCELLATION_INFORMATION: 'MultipliersDealCancellationInfo',
```

Comment:
```suggestion
    MULTIPLIERS_DEAL_CANCELLATION_INFO: 'MultipliersDealCancellationInfo',
```
or `MULTIPLIERS_CANCELLATION_INFO`, just to make it a bit shorter)